packages/test-app/src/Page.tsx

```diff
@@ -1,9 +1,33 @@
 import { useParams } from 'react-router-dom';
 import '@contentful/experience-builder-components/styles.css';
 import './styles.css';
-import { ExperienceRoot, useFetchBySlug } from '@contentful/experience-builder';
+import { ExperienceRoot, defineComponents, useFetchBySlug } from '@contentful/experience-builder';
 import { useContentfulClient } from './hooks/useContentfulClient';
 import { useContentfulConfig } from './hooks/useContentfulConfig';
+import Duplex from './components/Duplex';
```

Comment:
did you mean this change for the test-app to get in? We haven't really discussed it but my thoughts are the test-app would stay mostly unchanged unless its something for the app itself.

Reply:
Yeah I added it because we don't really have a way to test custom components from within the test-app. It's currently only the builtin components but I could remove it and or rename it to something that makes more sense

Reply:
@elylucas I tried refactoring this a bit to make more sense in a more generic use case as a test-app. Lmk your thoughts, I think eventually we'll want to add a basic DesignToken definition too so we can easily test design tokens.